src/Footer.scss

```diff
@@ -0,0 +1,6 @@
+.footer {
+  display: flex;
+  gap: 2em;
```

Comment:
Isn't this mixing techniques? I thought that grid and flex were two separate different ways of dealing with layout and that gap was grid

Reply:
flex and grid both use gap

Reply:
Oh cool, learn something new haha

Reply:
[grid, flex, and multi-column ](https://css-tricks.com/almanac/properties/g/gap/) layouts all use it